project/Dependencies.scala

```diff
@@ -8,7 +8,7 @@ object Dependencies {
   //libraries
   val sentryRavenLogback = "net.kencochrane.raven" % "raven-logback" % "6.0.0"
   val identityCookie = "com.gu.identity" %% "identity-cookie" % "3.44"
-  val identityPlayAuth = "com.gu.identity" %% "identity-play-auth" % "0.10"
+  val identityPlayAuth = "com.gu.identity" %% "identity-play-auth" % "0.12"
```

Comment:
Includes https://github.com/guardian/identity-play-auth/commit/5f8cbae48f to avoid class-initialisation problem.